app/views/propertyLinking/upload_result.scala.html

```diff
@@ -0,0 +1,160 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import actions.propertylinking.requests.LinkingSessionRequest
+@import binders.propertylinks.EvidenceChoices.EvidenceChoices
+@import binders.propertylinks.EvidenceChoices
+@import config.ApplicationConfig
+@import models.{EvidenceType, _}
+@import models.attachment.Attachment
+@import models.upscan.FileStatus.FileStatus
+@import models.upscan._
+
+
+@this(govukTag: GovukTag, govukSummaryList: GovukSummaryList, mainLayout: views.html.mainLayout, formWithCSRF: FormWithCSRF, govukButton: GovukButton, govukErrorSummary: GovukErrorSummary)
+
+@(
+        evidenceType: EvidenceType,
+        evidenceChoice: EvidenceChoices,
+        submissionId: String,
+        uploadedFiles: Map[String, UploadedFileDetails] = Map(),
+        status: FileStatus,
+        fileReference: Option[String],
+        optAttachment: Option[Attachment])(implicit request: LinkingSessionRequest[_], messages: Messages, config: ApplicationConfig)
+
+@attachmentFileName = @{
+    optAttachment.map(_.fileName).getOrElse("")
+}
+
+@attachmentDownloadLink = @{
+    optAttachment.flatMap(_.scanResult.flatMap(_.downloadUrl))
+}
+
+@attachmentFileNameHtml = {
+@if(attachmentDownloadLink.isDefined) {
+    <a class="govuk-link" href="@attachmentDownloadLink.get">@attachmentFileName</a>
+} else {
+    <a class="govuk-link" href=""></a>
+}
+}
+
+@title = @{
+    (request.ses.clientDetails, evidenceType) match {
+        case (Some(_), Lease) => {
+            messages("uploadEvidence.lease.client.title")
+        }
+        case (Some(_), License) => {
+            messages("uploadEvidence.license.client.title")
+        }
+        case (Some(_), RatesBillType) => {
+            messages("uploadRatesBill.show.client.title")
+        }
+        case (Some(_), _) => {
```

Comment:
is there a client equivalent for this? or is it the same as whats displayed for case (None, _)? Could probably just do a case (_,_) at the end if so

Reply:
Good spot, It appears to be the same, if im honest this is a straight copy from the upload.scala however I wonder if we should have content specific for clients for anything that falls in here-> case (Some(_), _)